microsoft-edge/extensions-chromium/getting-started/part1-simple-extension.md

```diff
@@ -16,7 +16,13 @@ The goal for this tutorial is to build a Microsoft Edge extension, starting with
 *   Adding icons.
 *   Opening a default pop-up dialog.
 
-If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1). The source code has been updated from Manifest V2 to Manifest V3.
+If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1).  The source code uses Manifest V3.
```

Comment:
```suggestion
If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1).  The source code uses Manifest V3.  To see how the source code was updated from MV2, see **History** in the Extensions repo.
```
Do we want to mention MV2 at all?  If so, here's a good way to frame it.  
@bhuvanapriyap 
Applies to both PRs.

Reply:
Part 2 in Extensions repo has always used mv3.  Closing.